### PR TITLE
Feature/ep consumable

### PR DESCRIPTION
**Pull request by nref-dan**
Add EP Boost items to the DB - Note that this patch will crash the syndicate shop if the server code is not updated due to the missing templates

### CODE REVIEW

**Review by MikeJeffers — CHANGES_REQUESTED** [review]
Patches to be reviewed go in ../Staging_Dev/
SQL in there (and not in a subfolder) is "staged" to later be bundled in a patch (which may be 1-N separate sql files/patches/projects)

**Review by MikeJeffers — DISMISSED** [review]

**Comment by MikeJeffers** [comment]
DO NOT MERGE
Until after Patch 8 on 12/30 - and @Bler reviews!

**Comment by MikeJeffers** [comment]
Tested and working with https://github.com/OpenPerpetuum/PerpetuumServer/pull/89

Please note these PR's depend on eachother!

**Comment by MikeJeffers** [comment]
Refactored to support : https://github.com/OpenPerpetuum/PerpetuumServer/pull/92
Tested locally ready for review

File: Staging_Dev/AddEPItemPatch.sql [hunk 1]
```diff
@@ -198,20 +206,28 @@ go
...
+
+
+
 -- Update the subscription procedure to accept a value
 -- Split into 2 parts, removal and addition
-if object_id('extensionSubscriptionStart', 'P') is not null
-	drop procedure extensionSubscriptionStart;
+if object_id('dbo.extensionSubscriptionStart', 'P') is not null
+	drop procedure dbo.extensionSubscriptionStart;
 go
```
Comment (nref-dan) [inline Staging_Dev/AddEPItemPatch.sql:229]: Duplicate the above  if statement here but for the opp.X procedure. This is to ensure there isn't a patch error trying to overwrite the procedure

File: Staging_Dev/AddEPItemPatch.sql [hunk 2]
```diff
@@ -246,22 +262,22 @@ go
...
-	drop procedure extensionSubscriptionExtend;
+if object_id('dbo.extensionSubscriptionExtend', 'P') is not null
+	drop procedure dbo.extensionSubscriptionExtend;
 go
 
-if object_id('getExtensionSubscription', 'P') is not null
-	drop procedure getExtensionSubscription;
+if object_id('dbo.getExtensionSubscription', 'P') is not null
+	drop procedure dbo.getExtensionSubscription;
 go
```
Comment (nref-dan) [inline Staging_Dev/AddEPItemPatch.sql:272]: Same here

**Review by Bler — APPROVED** [review]
approved

**Pull request merged**